packages/interactor/test/create-interactor.test.ts

```diff
@@ -474,6 +474,29 @@ describe('@bigtest/interactor', () => {
       await expect(TextField('Password', { enabled: false }).exists()).resolves.toBeUndefined();
     });
 
+    it('can override default values', async () => {
```

Comment:
I have just one last suggestion to make the test case name _very_ specific so those coming later know what's going on.
```suggestion
    it('the values in an assertion override the default values for a filter', async () => {
```